docs/guide/usage/touchlink.md

```diff
@@ -1,6 +1,6 @@
 # Touchlink
 
-**Important:** Touchlink **only** works for adapters based on a Texas Instruments chip (chips starting with CC, e.g. CC2652).
+**Important:** Only works with Zigbee Coordinator that has Touchlink enabled in firmware and support for touchlinking in [zigbee-herdsman adapter code](https://github.com/Koenkk/zigbee-herdsman).
```

Comment:
It would be helpful to indicate which coordinator support this, this is rather technical. 

Something like 

> Touchlink **only** works for adapters based on a Texas Instruments chip (chips starting with CC, e.g. CC2652) and EZSP coordinator with touchlink enabled in the firmware.

Reply:
OK, hope this update is better: **Important:** The touchlinking function **only** works with Zigbee Coordinator adapters based on a Texas Instruments ZNP adapters (TI chips starting with "CC", e.g. CC2652) and Silicon Labs EZSP adapters (Silabs chips starting with "EFR32", e.g. EFR32MG21) with touchlink enabled in the Zigbee Coordinator firmware.